internal/httpclient/upload.go

```diff
@@ -0,0 +1,71 @@
+package httpclient
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"path/filepath"
+)
+
+// constant variables for HTTP request
+const (
+	POST         = "POST"
```

Comment:
That one's redundant. Just use "POST" where needed or, if you want to use a constant, use `http.MethodPost`, since it already exists.

Reply:
Sure. I will use that.